### PR TITLE
 Add docker configuration for saiserver-brcm  

### PR DESCRIPTION
**Pull request by taoyl-ms**

### CODE REVIEW

File: dockers/docker-saiserver-brcm/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+FROM docker-base
+
+RUN apt-get update
+
+COPY ["/deps/libsaibcm*.deb","/deps/libopennsl_*.deb","/deps/python-sdk-api_*.deb", "/deps/iproute2_*.deb", "/deps/libthrift-0.9.2_*.deb", "/deps/libnl-3-200_*.deb", "/deps/libnl-genl-3-200_*.deb", "/deps/libnl-route-3-200_*.deb", "/deps/"]
+
+RUN dpkg_apt() { [ -f $1 ] && { dpkg -i $1 || apt-get -y install -f; } || return 1; }; \
+    dpkg_apt /deps/libopennsl_*.deb \
+    && dpkg_apt /deps/libsaibcm_*.deb     \
+    && dpkg_apt /deps/libthrift-0.9.2_*.deb    \
```
Comment (lguohan) [inline dockers/docker-saiserver-brcm/Dockerfile]: we are using libthrift-0.9.3 in this repo. Please change to 0.9.3

File: dockers/docker-saiserver-brcm/start.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+function clean_up {
+    service rsyslog stop
+}
+
+start_mlnx()
```
Comment (lguohan) [inline dockers/docker-saiserver-brcm/start.sh]: remove this function as it is not needed.

File: dockers/docker-saiserver-brcm/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,24 @@
+FROM docker-base
+
+RUN apt-get update
+
+COPY ["/deps/libsaibcm*.deb","/deps/libopennsl_*.deb","/deps/python-sdk-api_*.deb", "/deps/iproute2_*.deb", "/deps/libthrift-0.9.3_*.deb", "/deps/libnl-3-200_*.deb", "/deps/libnl-genl-3-200_*.deb", "/deps/libnl-route-3-200_*.deb", "/deps/"]
```
Comment (qiluo-msft) [inline dockers/docker-saiserver-brcm/Dockerfile]: /deps/libsaibcm_*.deb

Comment (qiluo-msft) [inline dockers/docker-saiserver-brcm/Dockerfile]: The source paths are actually relative in concept.

File: dockers/docker-saiserver-brcm/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    && dpkg_apt /deps/libnl-3-200_*.deb        \
+    && dpkg_apt /deps/libnl-genl-3-200_*.deb   \
+    && dpkg_apt /deps/libnl-route-3-200_*.deb
+
+COPY ["deps/saiserver", "start.sh", "/usr/bin/"]
+
+COPY ["profile.ini", "portmap.ini", "/etc/sai/"]
+
+## Clean up
+RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y
```
Comment (qiluo-msft) [inline dockers/docker-saiserver-brcm/Dockerfile]: Merge RUN commands into one to save docker image size.

**Pull request merged**